Sources/TuistKit/Services/EditService.swift

```diff
@@ -71,5 +86,17 @@ final class EditService {
         }
     }
 
-    private static var temporaryDirectory: AbsolutePath?
+    private func loadPlugins(at path: AbsolutePath) -> Plugins {
+        guard let config = try? configLoader.loadConfig(path: path) else {
+            logger.warning("Unable to load config manifest, fix and re-run to enable extra functionality.")
```

Comment:
Totally agree! I think this makes sense for a follow up for sure! I wonder how difficult it would be to just listen to changes in the `Config.swift` or `Plugin.swift` and reload the project accordingly? I'll look into it as a follow up! 👍🏼